### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by sam46** [sam46/Tachyon#1]

**Comment by bassdude5** [comment]
Looking for just unit tests or something else?

**Comment by sam46** [comment]
@bassdude5 yeah just unit test, I'll add continuous integration using Travis once they're available

**Comment by bassdude5** [comment]
JUnits ok? Ill give em a shot if so

**Comment by sam46** [comment]
@bassdude5 I haven't give in much thought, but how are gonna go about it? use a local server or something?

**Comment by bassdude5** [comment]
I'd suggest basic JUnits and Mockito to start with, anything not covered by them is then open to discussion on how to test it

**Comment by leahpleurodon** [comment]
Might I suggest jaCoCo as well

**Comment by sam46** [comment]
@leahpleurodon  @bassdude5 
Yeah alright you guys seem to have more experience with testing than I do... teach me something then!
Thanks! I'll be checking out your PRs in a couple days

**Comment by sam46** [comment]
Still requires tests for actual file downloads
